core/src/proof_generation/proof_manager.rs

```diff
@@ -156,39 +139,10 @@ impl ProofManager {
 
     /// Create a proof of `VALID WALLET CREATE`
     fn prove_valid_wallet_create(
-        fees: Vec<Fee>,
-        keys: PublicKeyChain,
-        randomness: Scalar,
+        witness: SizedValidWalletCreateWitness,
+        statement: SizedValidWalletCreateStatement,
     ) -> Result<ValidWalletCreateBundle, ProofManagerError> {
-        // Build an empty wallet and compute its commitment
```

Comment:
Why don't we do this anymore? At a high level seems like we still want a commitment to an empty wallet, is this handled in `singleprover_prove`?

Reply:
It's handled in the caller; the proof manager now only accepts the abstract witness and statement types which contain an empty wallet.